test/parallel/test-http-outgoing-renderHeaders.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const common = require('../common');
+const assert = require('assert');
+
+const outHeadersKey = require('internal/http').outHeadersKey;
+const http = require('http');
+const OutgoingMessage = http.OutgoingMessage;
+
+const outgoingMessage = new OutgoingMessage();
```

Comment:
I've changed the code to use block scoping, it seemed a dirty solution without it, thanks for the advice!